index.html

```diff
@@ -672,14 +678,20 @@ <h3>Controllers</h3>
 }
             </pre>
 
-            <p class="note" title="Authorization vs authentication">
-Note that authorization provided by the value of `controller` is
-separate from authentication as described in Section [[[#authentication]]].
-This is particularly important for key recovery in the cases of cryptographic key
-loss, where the [=subject=] no longer has access to their keys, or cryptographic
-key compromise, where the [=controller=]'s trusted third parties need to
-override malicious activity by an attacker. See [[[#security-considerations]]]
-for information related to threat models and attack vectors.
+            <p class="note" title="Authentication versus Authorization">
+Note that the definition of [=authentication=] is different from the definition
+of [=authorization=]. Generally speaking, [=authentication=] answers the
+question of "Who is this?" while [=authorization=] answers the question of "Are
+they allowed to perform this action?". The `authentication` property in this
```

Comment:
```suggestion
actions are they allowed to perform?". The `authentication` property in this
```

Reply:
I like the previous wording better, but would be OK with either.

Reply:
I prefer the previous wording as well because I think this suggestion fundamentally changes the question to be about getting a list of someone's authorized actions vs. asking if they are authorized for the specific action at hand. These aren't actually the same thing and if asking the former, the latter would just have to then be asked do the actual authorization check -- so, I think the existing wording captures this directness better.